app/Domain/Integrations/Events/ActivationReminderEmailSend.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations\Events;
+
+use Illuminate\Foundation\Events\Dispatchable;
+use Ramsey\Uuid\UuidInterface;
+
+final class ActivationReminderEmailSend
```

Comment:
This event indicates that the Reminder mail was sent, but it is the trigger to send them, so maybe something like `ActivationExpired` matches it better